level-1/삼총사.py

```diff
@@ -0,0 +1,14 @@
+def solution(number):
+    number = sorted(number)
+    answer = 0
+    total = 0
+    
+    for i in range(len(number)):
+        for j in range(len(number)):
+            for k in range(len(number)):
+                if (i != j) and (j != k) and (k != i) and (number[i] + number[j] + number[k] == 0):
```

Comment:
Think of more elegant way of writing this condition. FYI, wrapping condition with `( )` is not pythonic.